src/hotspot/cpu/arm/gc/shared/cardTableBarrierSetAssembler_arm.cpp

```diff
@@ -44,10 +44,6 @@
 void CardTableBarrierSetAssembler::gen_write_ref_array_post_barrier(MacroAssembler* masm, DecoratorSet decorators,
```

Comment:
Is it necessary to modify the `cardTableBarrierSetAssembler*` classes? I believe Shenandoah just uses the `shenandoahBarrierSetAssembler*` classes.

Reply:
Of course you're right! I'm reverting this change. Thanks.